projects/Mallard/src/hooks/use-net-info.tsx

```diff
@@ -138,7 +139,7 @@ export const createNetInfoResolver = () => {
 		 * resolved), so return that. We pass it through `assembleNetInfo`
 		 * because we don't expose our entire internal state.
 		 */
-		if (statePromise != null) {
+		if (statePromise) {
```

Comment:
statePromise should return a promise or null/undefined, so checking its truthy should do here

Reply:
Sounds good 👍